monitor/test.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Not a good idea to put a test file here. Better create one in your test and remove it after test. Also I recommend to list this file in the .gitignore